packages/wrangler/src/__tests__/mock-console.ts

```diff
@@ -19,16 +22,26 @@ const std = {
   },
 };
 
-export function mockConsoleMethods() {
+/**
+ * Mock out the console.xxx methods, capturing their calls via spies.
+ *
+ * The spies can be accessed via the returned object.
+ * Set the `columns` argument to ensure that Ink components are rendered in a suitably wide format.
+ */
+export function mockConsoleMethods({
+  columns = 500,
+}: { columns?: number } = {}) {
   beforeEach(() => {
     logSpy = jest.spyOn(console, "log").mockImplementation();
     errorSpy = jest.spyOn(console, "error").mockImplementation();
     warnSpy = jest.spyOn(console, "warn").mockImplementation();
+    process.stdout.columns = columns;
   });
   afterEach(() => {
     logSpy.mockRestore();
     errorSpy.mockRestore();
     warnSpy.mockRestore();
+    process.stdout.columns = originalColumns;
```

Comment:
I think this should be in a whole new method altogether. Let's not add it to this function. Maybe `setStdOutCoumns()`